src/org/tn5250j/framework/tn5250/tnvt.java

```diff
@@ -868,13 +868,15 @@ private void strpccmd()
 		try
 		{
 			int str = 11;
+			int offset = str+1;
+			int limit = 123; //TODO 1023 from V7R2
```

Comment:
123 is a magic number, please replace with meaningful  constant, e.g. static int PCCMD_MAX_LENGTH = 123